electron/models/Call.ts

```diff
@@ -86,4 +87,25 @@ export default class Call extends BaseEntity {
     )
     @JoinColumn({ name: 'suicide_risk_id' })
     suicideRisk: SuicideRisk;
+
+    @Column('varchar', { name: 'serial_number' })
+    serialNumber: String;
+
+    @Column('bigint', { name: 'date' })
+    Date: Date;
+
+    @Column('bigint', { name: 'time' })
+    Time: Timestamp;
+
+    @Column('bigint', { name: 'duration_in_seconds' })
+    DurationInMinutes: BigInteger;
+
+    @Column('varchar', { name: 'caller_name' })
+    CallerName: String;
+
+    @Column('varchar', { name: 'summary' })
+    Summary: String;
+
+    @Column('varchar', { name: 'note' })
+    Note: String;
```

Comment:
Follow the JavaScript property naming convention:

* `Date` -> `date`
* `Time` -> `time`
* etc.